README.md

```diff
@@ -12,18 +54,17 @@ A collection of starter bot templates used by [create-discordx](https://www.npmj
 | [4-music-player-ytdl](./4-music-player-ytdl)         | [@oceanroleplay](https://github.com/oceanroleplay) |
 | [5-music-player-lavalink](./5-music-player-lavalink) | [@oceanroleplay](https://github.com/oceanroleplay) |
 
-# 📜 Documentation
-
-> **[discord-ts.js.org](https://discord-ts.js.org)**
+## 📜 Documentation
 
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
 
-# ☎️ Need help?
+## ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Join and ask in our [Discord](https://discord-ts.js.org/discord)
 
-# Thank you
+## 💖 Thank you 💖
```

Comment:
I would like to keep, H1 heading format for all, so revert this as well.

Reply:
If I remember correctly, H1 is used as page heading, You wanna use H2 for non page headings.

Reply:
H2 is a sub heading. H2 is used for headings inside of H1

Reply:
Got ya, I will fix it to use H1 again then